packages/neuron-wallet/src/database/chain/entities/cell-local-info.ts

```diff
@@ -0,0 +1,35 @@
+import { Entity, PrimaryColumn, Column } from 'typeorm'
+
+export const outPointTransformer = {
```

Comment:
Usually, the `outPointTransformer` will be defined as a method of `CellLocalInfo` to make it cohesive, e.g.

```js
localCellInfo = LocalCellInfo.from(...)
infoList[localCellInfo.key]
```

By doing so, the transformer is embedded inside the schema and a clear key `key` is exposed instead.